### PR TITLE
Amanda Project Week 10

### PR DESCRIPTION
**Pull request by amandaflod**
Link to my deployed project: https://eloquent-snyder-48ed94.netlify.app/

My reflections on how this week's project turned out: I tried to focus more on the functionality and understanding what I was doing and less on the styling etc. I think it turned out really well!

### CODE REVIEW

File: code/src/summary.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.summary {
+  background-color: #8b95c9ff;
+  color: white;
+  margin-top: 50px;
+  width: 50%;
+  padding: 25px;
+  text-align: center;
```
Comment (moablomkvist) [inline code/src/summary.css:7]: Text that is longer than three lines can be difficult to read if it is centered. Try to think like books, they start from the margin on the left. Just a tip from a UX Writer. :)

File: code/src/summary.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './summary.css';
+
+const feelingResponse = feeling => {
```
Comment (moablomkvist) [inline code/src/summary.js:4]: Ambitious! Really good work Amanda.

**Review by moablomkvist — CHANGES_REQUESTED** [review]
Great work! Love the design and the ambitious summarys. Improve the responsiveness. :)

File: code/public/index.html [hunk 3]
```diff
@@ -1,34 +1,16 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link href="https://fonts.googleapis.com/css?family=Merriweather|Muli:300" rel="stylesheet">
+  <title>Technigo React App</title>
```
Comment (HIPPIEKICK) [inline code/public/index.html:8]: Don't forget to change the title :)

File: code/src/app.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
```
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/src/app.js:1]: Also, try to be consistent with double quotes or single quotes

[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/src/app.js:1]: And semicolons

File: code/src/summary.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  if (sleep === 'less than 5 hours') {
+    return 'Too little sleep can affect your mood in a negative way. If you have the time today, take a short nap on the couch to catch up on your sleep.'
+  } else if (sleep === '5-7 hours') {
+    return 'To up the energy for tomorrow, try to go to bed early tonight. It can also be good to stay away from any phone or computer screens for a little while before bedtime.'
+  } else if (sleep === '7-9 hours') {
+    return 'You seem to be sleeping well. If you don\'t already have a routine for bedtime, perhaps you should try it out in order to get enough sleep every night!'
+  } else {
+    return 'You probably have a lot of energy from a long night\'s sleep. Your body and mind will thank you for that.'
+  }
+}
```
Comment (HIPPIEKICK) [inline code/src/summary.js:38]: These could be broken out into their own components, or possibly just be switch statements. That's just a suggestion though

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Consistency in the usage of semicolons - should be used everywhere or not at all according to code guidelines.

**Pull request closed without merging**